### PR TITLE
Update introduction.stories.mdx

### PR DESCRIPTION
**Pull request by pseys**
Updating the theme introduction and removal of UI Toolkit references.

### CODE REVIEW

File: packages/theme/stories/introduction.stories.mdx [hunk 1]
```diff
@@ -2,33 +2,29 @@ import { Meta } from "@storybook/addon-docs";
...
+# Salt Theme 
 
-You can use styling attributes from the UITK theme to style many elements of your application - native HTML used for structural and layout purposes, your own custom components, etc.
+## Our approach
 
-### Explaining our approach
+The Salt design system is built from Vanilla CSS, which enables us to provide a custom and consistent set of styles. There are several benefits to using the Vanilla CSS framework: 
 
-For the Toolkit, our design system is built simply from vanilla CSS. There are a number of factors for this decision, including the following:
+- Salt has no third-party style dependencies, which means we are free to take our own approach to styling
```
Comment (libertymayc) [inline packages/theme/stories/introduction.stories.mdx:11]:
```suggestion
- Salt has no third-party style dependencies, which means we are free to take our own approach to styling.
```

**Review by libertymayc — APPROVED** [review]

File: packages/theme/stories/introduction.stories.mdx [hunk 2]
```diff
@@ -2,33 +2,29 @@ import { Meta } from "@storybook/addon-docs";
...
+## Our approach
 
-### Explaining our approach
+The Salt design system is built from Vanilla CSS, which enables us to provide a custom and consistent set of styles. There are several benefits to using the Vanilla CSS framework:
 
-For the Toolkit, our design system is built simply from vanilla CSS. There are a number of factors for this decision, including the following:
+- Salt has no third-party style dependencies, which means we are free to take our own approach to styling.
+- The Salt theme is built entirely using CSS custom properties.
+- Vanilla CSS provides performance improvements by removing the runtime overhead common in CSS-in-JS solutions.
+- Vanilla CSS allows the Salt theme to be published and consumed independent of the component library–It can be used in contexts where Salt components are not able to be used, e.g. static code deployment within a CMS.
```
Comment (sion-docs) [inline packages/theme/stories/introduction.stories.mdx:14]: Our style guide recommends against using 'e.g.'. Suggest either the obvious 'for example' or '...such as environments with static code deployed within a CMS.'

File: packages/theme/stories/introduction.stories.mdx [hunk 3]
```diff
@@ -41,100 +37,106 @@ A simple example will help to illustrate some of the key concepts:
...
+- **Component styles** - each component contributes its own stylesheet. Many of the CSS declarations reference CSS variables defined in the theme i.e. the variables will be defined at the theme level and used at the component level.
+
+\* Also includes one _utility_ class
+
+## Theme styles
+
+The Salt theme is a collection of design tokens split into a three-tier hierarchy. The token tiers are:
+
+1. **Foundation** - Foundation tokens are the primitive values for attributes like color, size, shadow and opacity.
+2. **Palette** - semantically grouped foundational colors–neutral, navigate, interact –which allow a consistent look-and-feel when used by characteristics, rather than referencing the foundational tokens directly. For example, actionable, selectable, and editable characteristic tokens all use the 'interact' palette. If the theme needed to change the background color of interactive elements it would only have to be updated in one place, the palette, instead of within each characteristic.
```
Comment (sion-docs) [inline packages/theme/stories/introduction.stories.mdx:54]: Numbering should be used for sequential steps, these should be bullets. Also there seems to be a bit of a mix-up at the start of the Palette description, maybe a copy and paste error?

**Comment by sion-docs** [comment]
Minor amends made directly in the commit.

File: packages/theme/stories/introduction.stories.mdx (hunk 2 above)
Comment (pseys) [reply packages/theme/stories/introduction.stories.mdx]: I've made the change, thanks.

File: packages/theme/stories/introduction.stories.mdx (hunk 3 above)
Comment (pseys) [reply packages/theme/stories/introduction.stories.mdx]: Thanks Sion, I've updated the copy.

**Review by sion-docs — DISMISSED** [review]

File: packages/theme/stories/introduction.stories.mdx [hunk 4]
```diff
@@ -2,33 +2,29 @@ import { Meta } from "@storybook/addon-docs";
...
 <Meta title="Documentation/Styles and Theming/Introduction" />
 
-## UI Toolkit styling
+# Salt Theme
 
-You can use styling attributes from the UITK theme to style many elements of your application - native HTML used for structural and layout purposes, your own custom components, etc.
+## Our approach
 
-### Explaining our approach
+The Salt design system is built from Vanilla CSS, which enables us to provide a custom and consistent set of styles. There are several benefits to using the Vanilla CSS framework:
```
Comment (joshwooding) [inline packages/theme/stories/introduction.stories.mdx:9]:
```suggestion
The Salt design system is built with Vanilla CSS, which enables us to provide a custom and consistent set of styles. There are several benefits to using Vanilla CSS:
```
```suggestion
The Salt design system is built using Vanilla CSS, which enables us to provide a custom and consistent set of styles. There are several benefits to using Vanilla CSS:
```

Either word is probably better, also vanilla CSS is not a framework

File: packages/theme/stories/introduction.stories.mdx [hunk 5]
```diff
@@ -41,100 +37,106 @@ A simple example will help to illustrate some of the key concepts:
...
-
-- CSS variables in the UI Toolkit theme are prefixed `--uitk-`. This is a rule with no exceptions.
+## CSS levels
+
+Salt styles are defined at two different levels, which operate together at runtime:
+
+- **Theme styles** - consisting of CSS Variables \*
+- **Component styles** - each component contributes its own stylesheet. Many of the CSS declarations reference CSS variables defined in the theme i.e. the variables will be defined at the theme level and used at the component level.
+
+\* Also includes one _utility_ class
```
Comment (joshwooding) [inline packages/theme/stories/introduction.stories.mdx:47]: Seems weird to mention this, I would probably omit it.

**Review by libertymayc — DISMISSED** [review]

**Review by libertymayc — APPROVED** [review]

**Review by libertymayc — APPROVED** [review]

**Pull request merged**